gen.py

```diff
@@ -12,16 +12,15 @@
 from io import BytesIO
 from json import load
 from os import listdir, makedirs, path, symlink
-from shutil import copy2, move, rmtree
+from shutil import copy2, rmtree
 from subprocess import PIPE, Popen, call
-
-from gi import require_version
-require_version('Rsvg', '2.0')
 try:
+    from gi import require_version
+    require_version('Rsvg', '2.0')
```

Comment:
Just to clarify, are we only checking for Rsvg being installed within the Cairo `if` block because we're assuming if they have Inkscape installed they'll have Rsvg?